feature-configs/cn-ran-overlays/all-in-one/example/sriov/sriov-network-node-policy-netdevice-mh.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-mh-dpdk-site-1-fqdn-worker1
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  isRdma: false
+  nicSelector:
+    pfNames: ["ens1f1#0-0"]
+  nodeSelector:
+    node-role.kubernetes.io/site-1-fqdn-worker1: ""
```

Comment:
This should be mentioned in the readme, same as the label below.
Also, it would make sense to have this labelling automated so we can CI this setup.

Maybe it makes sense to use the same worker-cnf selector?

Reply:
@josephdrichard

Reply:
Yes, we should use the worker-cnf label.  I'll update later today.

Reply:
@sabinaaledort please see above

Reply:
@josephdrichard, @fedepaol , @sabinaaledort 
Done in 4911f7